README.md

```diff
@@ -34,133 +32,74 @@ Composer support has been added,  Place the following lines in your composer.jso
 }
 ```
 
+Then run `composer install`
+
 **Note that it is necessary to include the vcs link to be sure that you are pulling from this repo.**
 
 The package manager for composer currently does **not** point to the correct repo for [clever/clever-php](https://packagist.org/packages/clever/clever-php).
```

Comment:
can we fix this?

Reply:
🤷‍♀️  It's been like that forever. May as part of a separate PR